### PR TITLE
Change seven display and update header

### PR DESCRIPTION
**Pull request by tcibils**
Hi @matteor6 , could you please check this pull request so we can merge it? Thanks!

### CODE REVIEW

**Comment by matteor6** [comment]
Can you please update the title to reflect all the changes?
Thanks,

File: Main/Main.ino [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /* 
  *  
- *  Creator : Thomas Cibils
- *  Last update : 05.02.2019
+ *  Creator : Thomas Cibils & Matteo Raviglione
```
Comment (matteor6) [inline Main/Main.ino:2]: Is this change needed by the client? Can you please link the JIRA ticket?
Thanks so much,

**Review by matteor6 — CHANGES_REQUESTED** [review]
Needs changes to comply to guidelines

File: Main/Main.ino (hunk 1 above)
Comment (tcibils) [reply Main/Main.ino:2]: Change made, please approve pull request

**Comment by matteor6** [comment]
Yes, good title, good quality, change plus rien, I say yes

**Comment by matteor6** [comment]
OMG J'AI MERGE, 1st time in my life
